src/Rules/NameUniqueForCities.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Lightit\Backoffice\City\Domain\Models\City;
+
+class NameUniqueForCities implements ValidationRule
+{
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        /** @phpstan-ignore-next-line */
```

Comment:
Here, you shoud type it as a City with PHPStan.

```suggestion
       /** @var City $city */
```

Add null if it can be; otherwise, should be like this

Reply:
The issue for the `@phpstan-ignore-next-line` was the use of `request()`